### ISSUE TITLE
Error in demo of README.md?

### ISSUE THREAD

**Issue by cachett-ML** [mlfoundations/open_clip#68]
Hi!

In the "Usage" part of the readme, we use model.encode_image() and model.encode_text() before computing the dot product of the features.

Hoewever those methods, by contrast with what is done during training,

https://github.com/mlfoundations/open_clip/blob/0d1127c05487e613f05275ace401f62a35a64e18/src/open_clip/model.py#L429

don't normalize the feature vectors.

Therefore it could bias the results. Am I wrong?

Best,
Théo

**Comment by rwightman** [comment]
Yeah demo in README, it's using the unormalized features via `encode_xxx` for the probs. @rom1504 is that intentional? 

The OpenAI Usage (https://github.com/openai/CLIP#usage) shows obtaining both the raw, unormalized features as here, but then uses normalized output for logits (via model forward).. although it's not calculating `image @ text.T` like our snippet.... 

FYI all use for training and eval / zero-shot in the repo is using normalized features, it's just this demo snippet.

**Comment by cachett-ML** [comment]
I can do a PR fixing this if it helps.
